src/main/java/com/example/securitynew/model/ShoppingCart.java

```diff
@@ -0,0 +1,37 @@
+package com.example.securitynew.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "shoppingCart")
+public class ShoppingCart {
+    @Id
+    @Column(name = "shoppingCart_id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long idShoppingCart;
```

Comment:
primary keys should be named like that
```suggestion
    private Long id;
```